reference/helpers_deprecated.go

```diff
@@ -0,0 +1,34 @@
+package reference
+
+import "github.com/distribution/reference"
+
+// IsNameOnly returns true if reference only contains a repo name.
+//
+// Deprecated: use [reference.IsNameOnly].
+func IsNameOnly(ref reference.Named) bool {
+	return reference.IsNameOnly(ref)
+}
+
+// FamiliarName returns the familiar name string
+// for the given named, familiarizing if needed.
+//
+// Deprecated: use [reference.FamiliarName].
```

Comment:
All functions / types were marked as deprecated, but continue to be functional, so consumers _can_ decide to continue using them, but the deprecation message should hopefully make them start preparing to move over to the new package (before switching to distribution v3 once a release for that that arrives)